i18n/locales/it/account-settings.json

```diff
@@ -0,0 +1,221 @@
+    },
+    "title": "Conferma l'eliminazione dell'account",
+    "warning-dialog": {
+      "close": {
+        "label": "ok"
+      }
+    },
+    "warnings": {
+      "cannot-merge": {
+        "text": "Non è ancora possibile unire l'account. Per prima cosa devi annullare tutti gli ordini di trading aperti e rimuovere tutte le risorse.",
+        "title": "Azione manuale richiesta"
+      }
+    }
+  },
+  "custom-trustline": {
+    "action": {
+      "cancel": "Annulla",
+      "trust": {
+        "long": "Fidati della moneta",
+        "short": "Fiducia"
+      }
+    },
+    "textfield": {
+      "code": {
+        "label": "Codice"
+      },
+      "issuer": {
+        "label": "Emittente",
+        "placeholder": "Rilascio della chiave pubblica dell'account"
+      },
+      "limit": {
+        "label": "Limite (opzionale)",
+        "placeholder": "Limita la fiducia in questa risorsa / saldo massimo da mantenere"
+      }
+    },
+    "title": "Aggiungi risorsa personalizzata"
+  },
+  "export-key": {
+    "action": {
+      "confirm": "Fatto",
+      "reveal": {
+        "long": "Rivela la chiave",
+        "short": "Fai clic per rivelare la chiave segreta"
+      }
+    },
+    "info": {
+      "backup": {
+        "title": "Esegui il backup della tua chiave segreta ora",
+        "paragraph-1": "Il backup della chiave segreta è l'unico modo per recuperare i tuoi fondi se dimentichi la password o non riesci più ad accedere al tuo dispositivo.",
+        "paragraph-2": "Annota la chiave o stampala. Conservala in un luogo sicuro e non condividerla con nessuno."
+      },
+      "export": {
+        "paragraph-1": "Il backup della chiave segreta è l'unico modo per recuperare i tuoi fondi se dimentichi la password o non riesci più ad accedere al tuo dispositivo.",
+        "paragraph-2": "Annota la chiave o stampala. Conservala in un luogo sicuro e non condividerla con nessuno."
+      },
+      "secret-key": "Annota la chiave su carta e conservala in un luogo sicuro.",
+      "tap-to-copy": "Tocca per copiare"
+    },
+    "textfield": {
+      "password": {
+        "label": "Password"
+      }
+    },
+    "title": {
+      "default": "Esporta la chiave segreta",
+      "initial-backup": "Backup della chiave segreta"
+    }
+  },
+  "manage-signers": {
+    "action": {
+      "add-co-signer": {
+        "long": "Aggiungi co-firmatario",
+        "short": "Firmatario"
+      },
+      "cancel": "Annulla",
+      "apply": {
+        "long": "Applica i cambiamenti",
+        "short": "Applica"
+      }
+    },
+    "signers-editor": {
+      "list": {
+        "item": {
+          "weight": "Peso",
+          "local-key": "Chiave locale"
+        }
+      },
+      "new-signer": {
+        "label": "Chiave Pubblica o indirizzo Stellar",
+        "placeholder": {
+          "long": "GABC… o indirizzo",
+          "short": "GABCDEFGHIJK… or alice*example.org"
```

Comment:
One last thing: `"long"` and `"short"` need to be swapped and "or" should probably be "o", i.e.:
```
"long": "GABCDEFGHIJK… o alice*example.org", 
"short": "GABC… o indirizzo",
```